packages/react-ape/renderer/reactApeRenderer.js

```diff
@@ -8,7 +8,13 @@
 
 import reconciler from 'react-reconciler';
 import reactApeComponent from './reactApeComponent';
-import {precacheFiberNode, updateFiberProps} from './reactApeComponentTree';
+import { precacheFiberNode, updateFiberProps } from './reactApeComponentTree';
+
+export type CanvasComponentContext = {
+  _renderQueueForUpdate: Array<mixed>,
+  type: 'canvas',
```

Comment:
type as string maybe?
or : `'canvas' || 'webgl`

Reply:
when we start supporting it we could do it,